todo_app/data/trello_items.py

```diff
@@ -0,0 +1,56 @@
+import requests
+import os
+
+from todo_app.data.Item import Item
+
+def add_item(title):
+    print(title)
```

Comment:
This sort of print can be useful for debugging, but is a really easy potential source of sensitive data leaking into your logs, e.g. to-dos like "send email to <address>", or "go see GP about <x>"; typically we want to keep sensitive data out of the logs so that we can happily share production logs around a wider set of people than we'd necessarily give access to sensitive production data, so a good habit to clear this out